src/popup/pages/AccountLogin/index.tsx

```diff
@@ -16,7 +16,8 @@ interface IProps {
 class AccountLogin extends Component<WithStyles & IProps, {}> {
 
   public componentDidMount() {
-    this.props.store.accountLoginStore.getAccounts();
+    const andValidateNetwork = true;
+    this.props.store.accountLoginStore.getAccounts(andValidateNetwork);
```

Comment:
`this.props.store.accountLoginStore.getAccounts(true);`?

Reply:
I was thinking it would be easier to understand what the true parameter is referring to by giving it a var name, but was also on the fence about whether it made more sense to just collapse it into 1 line since you can also easily find out what true refers to just by looking at the method definition. Seems you prefer the later, ill change it.